apps/experiments/tests/test_models.py

```diff
@@ -543,9 +536,9 @@ def test_create_new_source_material_version(self):
 @pytest.mark.django_db()
 class TestExperimentRouteVersioning:
```

Comment:
This test can be removed as well?

Reply:
Same with TestExperimentRoute

Reply:
Yes, it can all go but I had to pick somewhere to stop with this PR so this is it for now.

Reply:
I figured this might have been the reason :+1: